docs/building_blocks_for_flexible_content.md

```diff
@@ -320,16 +325,28 @@ A hero block renders a vertical column of blocks in a two-thirds-width block on
     blocks:
       - type: heading
         content: Rorem ipsum dolor sit
-      - type: govspeak
-        content: |
-          <p>Yorem ipsum dolor sit amet, consectetur
-          adipiscing elit. Nunc vulputate libero et velit
-          interdum, ac aliquet odio mattis class.</p>
-      - type: action_link
-        text: "Learn more about our goals"
-        href: "/landing-page/goals"
 ```
 
+When authored in Whitehall, you should upload the images and then refer to them using their markdown syntax as follows:
+
+```yaml
+- type: hero
+  image:
+    alt: "Placeholder alt text"
```

Comment:
I thought we agreed no alt text? Hero should always be decor so we shouldn't supply this.

Reply:
Yep, this needs removing from the example - sorry!

Reply:
Fixed in 6e09625b8532b9235a9a105c960abda5a453ab93